app/tornado_handlers/download.py

```diff
@@ -155,6 +154,75 @@ def kml_colors(flight_mode):
+            ulog = load_ulog_file(log_file_name)
+            param_keys = sorted(ulog.initial_parameters.keys())
+
+            self.set_header("Content-Type", "text/plain")
+            self.set_header('Content-Disposition', 'inline; filename=vehicle.params')
+
+            delimiter = '	'
+            for param_key in param_keys:
+                self.write("1")
+                self.write(delimiter)
+                self.write("1")
+                self.write(delimiter)
+                self.write(param_key)
+                self.write(delimiter)
+                self.write(str(ulog.initial_parameters[param_key]))
+
+                #if the value is an int write a 6, if not write a 9
+                if(type(ulog.initial_parameters[param_key]) == int):
+                    self.write(delimiter)
+                    self.write("6")
+                else:
+                    self.write(delimiter)
+                    self.write("9")
+
+                self.write('\n')
+
+        elif download_type == '5': # download non-default parameters for QGC
+            ulog = load_ulog_file(log_file_name)
+            param_keys = sorted(ulog.initial_parameters.keys())
+
+            self.set_header("Content-Type", "text/plain")
+            self.set_header('Content-Disposition', 'inline; filename=vehicle-nondefault.params')
+            delimiter = '	'
+
+            # Use defaults from log if available
+            if ulog.has_default_parameters:
+                system_defaults = ulog.get_default_parameters(0)
+                airframe_defaults = ulog.get_default_parameters(1)
+                for param_key in param_keys:
+                    try:
+                        param_value = ulog.initial_parameters[param_key]
+                        is_default = True
+                        if param_key in airframe_defaults:
+                            is_default = param_value == airframe_defaults[param_key]
+                        elif param_key in system_defaults:
+                            is_default = param_value == system_defaults[param_key]
+
+                        if not is_default:
+                            self.write("1")
+                            self.write(delimiter)
+                            self.write("1")
+                            self.write(delimiter)
+                            self.write(param_key)
+                            self.write(delimiter)
+                            self.write(str(param_value))
+
+                            #if the value is an int write a 6, if not write a 9
+                            if(type(param_value) == int):
+                                self.write(delimiter)
+                                self.write("6")
+                            else:
+                                self.write(delimiter)
+                                self.write("9")
+
+                            self.write('\n')
+                    except:
+                        pass
+
             else:
```

Comment:
This `else` case was part of the `elif download_type == '3': # download the non-default parameters` case above and needs to be copied up to that case again, and here it needs to be adjusted for the QGC format.